include/zrepl_mgmt.h

```diff
@@ -180,9 +186,9 @@ extern int uzfs_zinfo_init(void *zv, const char *ds_name,
 extern zvol_info_t *uzfs_zinfo_lookup(const char *name);
 extern void uzfs_zinfo_replay_zil_all(void);
 extern int uzfs_zinfo_destroy(const char *ds_name, spa_t *spa);
-uint64_t uzfs_zvol_get_last_committed_io_no(zvol_state_t *zv);
+uint64_t uzfs_zvol_get_last_committed_io_no(zvol_state_t *zv, char *key);
 void uzfs_zvol_store_last_committed_io_no(zvol_state_t *zv,
-    uint64_t io_seq);
+    uint64_t io_seq, char *key);
```

Comment:
lets make this as (key, value) rather than (val, key)

Reply:
ok. I will try to cover this change in the next PR.